pkg/ebpf/events_pipeline.go

```diff
@@ -569,6 +569,14 @@ func (t *Tracee) sinkEvents(ctx context.Context, in <-chan *trace.Event) <-chan
 				continue
 			}
 
+			// is the policy globally disabled?
+			event.MatchedPoliciesUser = t.policiesManager.Match(event.MatchedPoliciesUser)
```

Comment:
A note to my future self @geyslan: this just drops disabled events at the final stage of the pipeline. As discussed offline, it will be replaced by a mechanism that also updates kernel space. Probably the policies versioning.